DXRFixes/DeusEx/Classes/ElevatorMover.uc

```diff
@@ -23,17 +66,23 @@ function SetSeq(int seqnum)
     local int prevKeyNum;
     local float dist;
 
+    Initialize();
+
     if( MoveTime/2 < Level.TimeSeconds-lastTime )
         oldSeq = true;
 
     if ( bIsMoving && !oldSeq )
         return;
 
+    dist = VSize(BasePos + KeyPos[seqnum] - Location);
+
+    if (numKeyPos > 2)
+        MoveTime = FMax(dist / moveSpeed, originalMoveTime / 2.5);
```

Comment:
well the speed isn't consistent, and it slows down as you press the button over and over again, but it might be complicated to fix that issue because I noticed the vandenberg cmd elevator can be a bit janky without this
```suggestion
        MoveTime = dist / moveSpeed;
```

idk if we want to try to get something truly consistent like that working for every elevator
```suggestion
        MoveTime = FMax(dist / moveSpeed, originalMoveTime / 2.5);
```

Reply:
Elevators feel like they move way too fast between short stops if you don't put any restriction on it like this. I think it feels pretty nice the way I have it right now. There are only a few places it has an effect.

Reply:
That was where my suggestion the other day about having a further adjustment on the moveSpeed when it's initially calculated (eg. half or 2/3rds of the calculated speed or something) comes into play - finding what speed feels right based on the calculated values we can derive

Reply:
I think it's fine too, at least for now, just want to point out that the speed is actually not consistent in these cases, but much better than vanilla

Reply:
> That was where my suggestion the other day about having a further adjustment on the moveSpeed when it's initially calculated (eg. half or 2/3rds of the calculated speed or something) comes into play - finding what speed feels right based on the calculated values we can derive

the problem is not the MoveTime, you can calculate that mathematically perfectly, but underneath it the Mover code does some interpolation for acceleration/deceleration